libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewApi.kt

```diff
@@ -112,4 +113,9 @@ abstract class NavigationViewApi {
      * Enable/Disable replay trip session based on simulated locations.
      */
     abstract fun routeReplayEnabled(enabled: Boolean)
+
+    /**
+     * An object to play voice instructions.
+     */
+    abstract val voiceInstructionsPlayer: MapboxVoiceInstructionsPlayer?
```

Comment:
That way, this will also work for systems like Android Auto

Reply:
> The concept of DropInUi is to have things work without custom implementations.

@kmadsen afaik we have feature request from a customer to allow playing customers voice messages
@abhishek1508 could you confirm?

Reply:
This is a potential solution, but won't be scalable. We have seen with different customers that they have the need to play a random voice instruction in addition to the regular guidance instruction played by the player. Just enabling/disabling this feature will solve the problem, but will not be applicable with other requests. Exposing `voiceInstructionPlayer` allows customers to play any kind of instructions they want associated with any kind of `RoadObjects` (not only incidents). For ex: It's very common for navigation apps to play voice instructions like `There is a heavy traffic congestion ahead. You are still on the fastest route`. 

It would become very difficult to enable every single capability with a flag such as `enableIncidentInstructions(true)`.

Reply:
Then we need a different way to expose the voiceInstructionsPlayer. For example, through the MapboxAudioGuidance. That way you can access the service when the NavigaitonView is out of scope.

Similar to how you can access MapboxNavigation through MapboxNavigationApp, or how you can prepare a MapboxMap before launching the NavigationView. 

That way these customizations are supported for other SDKs like android auto.